3.11/indexing-inverted.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: default
+description: >-
+  You can use inverted indexes to speed up a broad range of AQL queries,
+  from simple to complex, including full-text search
+---
+# Inverted indexes
+
+{{ page.description }}
+{:class="lead"}
+
+Documents hold attributes, mapping attribute keys to values.
+Inverted indexes store mappings from values (of document attributes) to their
+locations in collections. You can create these indexes to accelerate queries
+like value lookups, range queries, accent- and case-insensitive search,
+wildcard and fuzzy search, nested search, search highlighting, as well as for
+sophisticated full-text search with the ability to search for words, phrases,
+and more.
+
+You can use inverted indexes stand-alone in `FILTER` operations of AQL queries,
+or add them to [Search Alias Views](arangosearch.html#getting-started-with-arangosearch)
+to search multiple collections at once and to rank search results by relevance.
+
+## Defining inverted indexes
+
+Inverted indexes are defined per collection. You can add an arbitrary number of
+document attributes to each index. Every attribute can optionally be processed
+with an [Analyzer](analyzers.html), for instance, to tokenize text into words.
+
+### Basic definition
+
+For example, you can create an inverted index for the attributes `value1` and
+`value2` with the following command in arangosh:
+
+```js
+db.<collection>.ensureIndex({ type: "inverted", fields: ["value1", "value2"] });
+```
+
+### Processing fields with Analyzers
+
+The fields are processed by the `identity` Analyzer by default, which indexes
+the attribute values as-in. To define a different Analyzer for the second field,
+you can pass an object with the field settings instead. It also allows you to
+overwrite the Analyzer features:
```

Comment:
I think my sentences aren't clear enough because the idea was to say "you can set an Analyzer for a field" and in the last sentence "you can also set the features".